### PR TITLE
Make an exception for spurious-executable-perm.

### PR DESCRIPTION
**Pull request by marxin**
Except files that live in %docdir/examples.

### CODE REVIEW

**Comment by marxin** [comment]
It's covering `docdata-examples.diff` in #292.

File: rpmlint/descriptions/FilesCheck.toml [hunk 1]
```diff
@@ -123,6 +123,8 @@ spurious-executable-perm="""
 The file is installed with executable permissions, but was identified as one
 that probably should not be executable. Verify if the executable bits are
 desired, and remove if not.
+NOTE: example scripts should be packaged under %docdir/examples, which will avoid
```
Comment (Conan-Kudo) [inline rpmlint/descriptions/FilesCheck.toml]: Isn't this `%{_docdir}`?

File: test/test_files.py [hunk 2]
```diff
@@ -149,3 +149,12 @@ def test_lib_regex():
...
          '/usr/lib64/rsocket/binary',))
+
+
+@pytest.mark.parametrize('package', ['binary/executableperm'])
+def test_executable_permission(tmpdir, package, filescheck):
+    output, test = filescheck
+    test.check(get_tested_package(package, tmpdir))
+    out = output.print_results(output.results)
+    assert 'W: spurious-executable-perm /usr/share/doc/packages/bwidget/Widget.html' in out
+    assert 'W: spurious-executable-perm /usr/share/doc/packages/bwidget/script/Widget2.html' not in out
```
Comment (Conan-Kudo) [inline test/test_files.py]: Ugh, please don't allow `/usr/share/doc/packages` to enter upstream rpmlint. Nobody else does such a boneheaded modification.

**Comment by Conan-Kudo** [comment]
@marxin I'm not sure if examples should be permitted to be executable, though... Why not just install them as regular binaries, then?

File: test/test_files.py (hunk 2 above)
Comment (marxin) [reply test/test_files.py]: What wrong with that? It's only a test-case that I added here..

**Comment by Conan-Kudo** [comment]
I'm inclined to say this is not okay at all. If random binary files are in `/usr/share/doc` then that's a problem.

**Comment by marxin** [comment]
> I'm inclined to say this is not okay at all. If random binary files are in `/usr/share/doc` then that's a problem.

I'm fine with that.
@scarabeusiv: please check the patch in #292

**Pull request closed without merging**